### PR TITLE
Fixup networks

### PR DESCRIPTION
**Pull request by jovial**
- Use correct VLAN segmentation ids
- Add the public network

### CODE REVIEW

File: etc/sib-config/sib-config.yml [hunk 1]
```diff
@@ -59,6 +59,32 @@ sib_networks:
...
+  # Subnet configuration.
+  subnets:
+    - "{{ sib_subnet_public }}"
+
+# Public subnet.
+sib_subnet_public:
+  name: "{{ sib_network_public_name }}"
+  cidr: "10.12.9.0/24"
+  gateway_ip: "10.12.9.1"
+  allocation_pool_start: "10.12.9.10"
```
Comment (oneswig) [inline etc/sib-config/sib-config.yml]: Can you run from 100-199 here, that doesn't leave much wriggle room, and I like nice round numbers...

File: etc/sib-config/sib-config.yml [hunk 2]
```diff
@@ -59,6 +59,32 @@ sib_networks:
...
+  allocation_pool_end: "10.12.9.127"
+  # host_routes:
```
Comment (oneswig) [inline etc/sib-config/sib-config.yml]: This commented out host route looks oddly familiar...

File: etc/sib-config/sib-config.yml [hunk 3]
```diff
@@ -59,6 +59,32 @@ sib_networks:
...
+  #   - destination: "10.4.0.0/16"
+  #     nexthop: "10.60.253.255"
 
 # SIB internal network name.
 sib_network_internal_name: "internal"
```
Comment (oneswig) [inline etc/sib-config/sib-config.yml:137]: Do we need this defining for Ironic?  The subnet pool seems a little odd and all.

**Review by oneswig — CHANGES_REQUESTED** [review]
Nice job, just a couple of questions.

File: etc/sib-config/sib-config.yml (hunk 3 above)
Comment (jovial) [reply etc/sib-config/sib-config.yml:137]: Seems we need this for routing from the provisioning network to the internal

**Pull request merged**